.travis.yml

```diff
@@ -26,3 +26,8 @@ env:
   - VARIANT=foundation-layout GENERATOR_INPUT="example.com\nstaging.example.com\nY\nY\nY\nn"
 script:
   - ./template-test/ci-run.sh
+notifications:
+  slack:
+    if: branch = master
+    on_success: change
```

Comment:
I have changed this so that it only notifies if the build status of master **changes** - this should keep it quiet most of the time.